navigation/AppNavigator.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
-import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import React from "react";
+import { createAppContainer, createSwitchNavigator } from "react-navigation";
 
-import MainTabNavigator from './MainTabNavigator';
+import CheckoutScreen from "../screens/CheckoutScreen";
+import ClerkProductsScreen from "../screens/ClerkProductsScreen";
+import CustomerPhoneNumberScreen from "../screens/CustomerPhoneNumberScreen";
+import MainTabNavigator from "./MainTabNavigator";
 
 export default createAppContainer(
   createSwitchNavigator({
     // You could add another route here for authentication.
     // Read more at https://reactnavigation.org/docs/en/auth-flow.html
     Main: MainTabNavigator,
+    CustomerPhoneNumberScreen: CustomerPhoneNumberScreen,
```

Comment:
Could you add a "Future work" section and note that we didn't have time to figure out whether to change this to a StackNavigator?

Reply:
Added a TODO!

Reply:
Ty (: